### ISSUE TITLE
Apple Pencil contact on iPad with Listener widget is delayed

### ISSUE THREAD

**Issue by matthew-carroll** [flutter/flutter#109022]
I'm working on a drawing app. Quick stylus response is critical.

I've found that using an Apple Pencil on an iPad results in a delayed call to `onPointerDown` when using a `Listener` widget.

When I try the same experiment on a Samsung tablet with a Samsung stylus, the delay **doesn't** happen.

When I try touching the iPad screen with my finger, the delay **doesn't** happen.

I'm only able to cause the delay with the Apple Pencil on the iPad.

The experiment can be repeated with this app code:

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(const App());
}

class App extends StatefulWidget {
  const App({Key? key}) : super(key: key);

  @override
  State<App> createState() => _AppState();
}

class _AppState extends State<App> {
  @override
  Widget build(BuildContext context) {
    return Listener(
      onPointerDown: (_) {
        print("Pointer down!");
      },
      child: Container(
        color: Colors.green,
      ),
    );
  }
}
```

Flutter version:
```
Flutter 3.0.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision f1875d570e (3 weeks ago) • 2022-07-13 11:24:16 -0700
Engine • revision e85ea0e79c
Tools • Dart 2.17.6 • DevTools 2.12.2
```

iPad info:
 * iPad Pro (12.9 inch) (from 2018 or 2019, can't remember)
 * iOS 15.1

**Comment by goderbauer** [comment]
cc @fbcouch @LongCatIsLooong

**Comment by fbcouch** [comment]
Maybe a duplicate of https://github.com/flutter/flutter/issues/101016 ? I don't think the fix has made it to stable yet

**Comment by matthew-carroll** [comment]
@fbcouch I don't know the internals of the changes related to that other ticket, but I'm not using any OS features in this case. I haven't enabled "scribble", if that's something that a user would enable. I'm just processing pointer events that happen to come from a stylus.

**Comment by Run10Break1** [comment]
I also have the same problem with "onPanUpdate" on GestureDectector.
there is a slight gap to recognize the first touch when using an apple pencil.
I wonder if there is any update.

**Comment by fbcouch** [comment]
Can you try it on the the new stable version that just dropped (3.3.0)? That has the fix that I mentioned previously that sounds like it might be the same issue.

**Comment by Run10Break1** [comment]
Oh, that's fixed like magic, thank you!

**Comment by matthew-carroll** [comment]
I've confirmed that this issue is resolved on my end with Flutter 3.3.3. I'm now getting much quicker response for drawing with an Apple pencil in Flutter.
